network/network_windows.go

```diff
@@ -185,6 +190,61 @@ func (nm *networkManager) newNetworkImplHnsV1(nwInfo *NetworkInfo, extIf *extern
 	return nw, nil
 }
 
+// add ipv4 and ipv6 routes (if dual stack mode) to windows Node
+// in dualstack mode, pods are created from different subnets on different nodes, gateway has to be node ip if pods want to communicate with each other
+// add routes to make node understand pod IPs come from different subnets and VFP will take decisions based on these routes to forward traffic and avoid Natting
+func (nm *networkManager) addNewNetRules(nwInfo *NetworkInfo) error {
+	var (
+		err error
```

Comment:
why do we need to declare these here upfront?

Reply:
use same format as appIPV6RouteEntry() here